packages/forms-web-app/src/dynamic-forms/s78-questionnaire/journey.js

```diff
@@ -30,154 +33,131 @@ class S78Journey extends Journey {
 			journeyTitle
 		);
 
+		const meetsCondition = meetsConditionBuilder(response, questions);
+		const meetsConditions = meetsConditionsBuilder(response, questions);
+
 		this.sections.push(
 			new Section('Constraints, designations and other issues', 'constraints')
 				.addQuestion(questions.appealTypeAppropriate)
 				.addQuestion(questions.changesListedBuilding)
 				.addQuestion(questions.changedListedBuildingNumber)
-				.withCondition(
-					response.answers && response.answers[questions.changesListedBuilding.fieldName] == 'yes'
-				)
+				.withCondition(meetsCondition('changesListedBuilding', 'yes'))
```

Comment:
Looks great - my only thought/question.
Did you consider passing in the question rather than just a string? Makes it slightly longer, but we also (in theory) get autocomplete. 

e.g. `meetsCondition(questions.changesListedBuilding, 'yes')`

Reply:
thanks - have tweaked it to match your suggestion!